### PR TITLE
Add Powers Family Medicine

### PR DESCRIPTION
**Pull request by KagurazakaHinagi**
1. Add Powers Family Medicine, an HRT provider in the United States. 
2. Collapse US HRT pages.
3. Clarify the estradiol pill could be either orally or sublingually taken.

### CODE REVIEW

**Comment by mofummofu** [comment]
境外看诊大概率会有法律问题

File: content/zh-cn/docs/hrt/us/powers.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Powers Family Medicine
+slug: powers
+---
+
+网站: <https://powersfamilymedicine.com/>
```
Comment (ArtsEpiphany) [inline content/zh-cn/docs/hrt/us/powers.md:6]:
```suggestion
网站: <https://powersfamilymedicine.com>
```

File: content/zh-cn/docs/hrt/us/powers.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Reddit: <https://www.reddit.com/r/DrWillPowers/>
```
Comment (ArtsEpiphany) [inline content/zh-cn/docs/hrt/us/powers.md:8]:
```suggestion
Reddit: <https://www.reddit.com/r/DrWillPowers>
```

File: content/zh-cn/docs/hrt/us/powers.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+1. 诊所工作人员发电子邮件联系患者
+1. 补充证件和保险卡信息
+1. 选择医生/护士（Dr. Will Powers 本人排队时间较长）
+1. 排队等待
+1. 确认首诊时间
+1. 线上/线下面诊
+
+## 远程就诊的注意事项
+
+1. 部分州不支持远程看诊，[详见](https://www.reddit.com/r/DrWillPowers/wiki/covid/telemed)
```
Comment (ArtsEpiphany) [inline content/zh-cn/docs/hrt/us/powers.md:34]:
```suggestion
1. 部分州不支持远程看诊，详见<https://www.reddit.com/r/DrWillPowers/wiki/covid/telemed>
```

**Review by ArtsEpiphany — COMMENTED** [review]
感谢提交pr！

**Pull request merged**